sci-physics/pyhepmc/pyhepmc-2.11.0.ebuild

```diff
@@ -0,0 +1,17 @@
+EAPI=8
+
+PYTHON_COMPAT=( python3_{10..12} )
+DISTUTILS_USE_PEP517=setuptools
+inherit distutils-r1 pypi
+
+DESCRIPTION="A Pythonic wrapper for the HepMC3 C++ library."
```

Comment:
Should we RDEPEND on hepmc?

Reply:
No it compiles it by itself at a fixed version through some python build system. But i should add a BDEPEND on cmake.

Reply:
It bundles hepmc? Is it feasible to make it use the system version instead?

Reply:
That makes sense